pennylane/tape/tapes/tape.py

```diff
@@ -814,6 +818,32 @@ def graph(self):
 
         return self._graph
 
+    @property
+    def resources(self):
+        """Resource requirements of a quantum circuit.
+
+        Returns:
+            dict[Operation : int]: how many times the constituent Operations are applied
+        """
```

Comment:
It would be good to add a minimal usage example here, showing expected output! See https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#code-examples

Reply:
Done, should we add the properties [here](https://pennylane.readthedocs.io/en/stable/code/qml_tape.html) as well? They are not auto-included by sphinx with the other methods.

Reply:
Looks like it worked! https://pennylane--862.org.readthedocs.build/en/862/code/api/pennylane.tape.QuantumTape.html#pennylane.tape.QuantumTape.resources

Reply:
Whoops, was looking at the live version of the site - no wonder I didn't see them.